src/OperationalInsights/OperationalInsights/Models/PSCluster/PSCapacityReservationProperties.cs

```diff
@@ -11,13 +11,27 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-namespace Microsoft.Azure.Commands.OperationalInsights.Models
+using Microsoft.Azure.Management.OperationalInsights.Models;
+
+namespace Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster
 {
-    public class UpdatePSTableParameters : OperationalInsightsParametersBase
+    public class PSCapacityReservationProperties
     {
-        public string TableName { get; set; }
-        //public bool? IsTroubleshootEnabled { get; set; }//in the next API version i.e 2020-10-01
-        public int? RetentionInDays { get; set; }
 
+        /// <summary>
+        /// The last time Sku was updated.
+        /// </summary>
+        public string LastSkuUpdate { get; private set; }
+
+        /// <summary>
+        /// CapacityReservation value in GB.
+        /// </summary>
+        public long? MinCapacity { get; private set; }
+
```

Comment:
PSCluster is modules the same as the SDK, so the Cluster.Sku contains the capacity and SKU name, while Cluster.CapacityReservationProperties contains MinCapacity and LastSkuUpdate - the same way as the PS classes

Reply:
no, the purpose I asked you to create this class is to avoid breaking change where MaxCapacity got removed in your swagger. Please do what I suggested.

Reply:
@VeryEarly 
PSClusterSku.cs will become redundant if I will move Capacity out and into PSCapacityReservationProperties.cs.
can i delete PSClusterSku.cs and move the sku name out and to PSCluster.cs or to PSCapacityReservationProperties.cs
?

Reply:
@VeryEarly ping

Reply:
@BethanyZhou  tagging you here as you asked me to reverse the change that was asked by @VeryEarly